frontend/js/src/components/HorizontalScrollContainer.tsx

```diff
@@ -0,0 +1,116 @@
+import {
+  faChevronLeft,
+  faChevronRight,
+} from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { throttle } from "lodash";
+import React, { PropsWithChildren } from "react";
+import { useDraggable } from "react-use-draggable-scroll-safe";
+
+type HorizontalScrollContainerProps = {
+  showScrollbar?: Boolean;
+  enableDragScroll?: Boolean;
+  scrollContainerCssClass?: string;
```

Comment:
Let's change the name of props to className
```suggestion
  className?: string;
```